jupyter-book/_static/default_text_env_setup.md

```diff
@@ -0,0 +1,28 @@
+1. **Save the yml content**:
```

Comment:
There might need to be another step before that which explains how to install Conda. Could you add one sentence on that, please?